### DISCUSSION TITLE
WebRTC with VP8 instead of H264

### DISCUSSION THREAD

**Discussion by abaumhof** [blakeblackshear/frigate#8831] [General]
Hi, I'm using Frigate and it works beautifully. I have a Sonoff NSPanel Pro sideloaded with the homeassistant companion app and/or wallpaper. I'm having trouble viewing the video feed on the device. I think H264 is only supported on Android if there is hardware accelleration for it and I kind of think that that's not the case... I can hear audio, but no video... I think VP8/9 should work, but I'm having trouble configuring go2rtc to do the conversion... Is that easily possible with some ffmpeg magic? Thanks for your help.

**Comment by NickM-27** [comment]
the browser must support it, I do not know any browser that supports VP8 over WebRTC
